ntc_templates/templates/cisco_ios_show_tacacs.textfsm

```diff
@@ -24,6 +24,9 @@ Start
   ^\s+Failed\s+Connect\s+Attempts:\s+${FAILED_CONNECTIONS}\s*$$
   ^\s+Total\s+Packets\s+Sent:\s+${PACKETS_SENT}\s*$$
   ^\s+Total\s+Packets\s+Recv:\s+${PACKET_RECEIVED}\s*$$
+  ^\s+Server\s+Status:.+?\s*$$
```

Comment:
```suggestion
  ^\s+Server\s+Status:\s+\S+\s*$$
```

From what we have seen at this point there's a space `\s+` followed by a word `\S+`.